plugins/callback/diy.py

```diff
@@ -0,0 +1,1421 @@
+        {% set prefix = '  ' %}
+        {% set suffix = '  ' %}
+        {% set color_two = lookup('vars','bright_purple') %}
+        {% elif key == 'skipped' %}
+        {% set color_one = lookup('vars','cyan_bg_black_fg') %}
+        {% set prefix = '   ' %}
+        {% set suffix = '   ' %}
+        {% set color_two = lookup('vars','cyan') %}
+        {% else %}
+        {% set color_one = "" %}
+        {% set prefix = "" %}
+        {% set suffix = "" %}
+        {% set color_two = "" %}
+        {% endif %}
+        {{ color_one }}{{ "%s%s%s" | format(prefix,key,suffix) }}{{ color_two }}: {{ ansible_callback_diy.stats[key] | to_nice_yaml }}
+        {% endif %}
+        {% endfor %}
+
+    tasks:
+      - name: custom banner with default plugin result output
+        debug:
+          msg: "default plugin output: result example"
+
+      - name: override from task vars
+        debug:
+          msg: "example {{ two }}"
+        changed_when: true
+        vars:
+          white_fg_red_bg: "\e[0m\e[48;5;1m"
+          two: "{{ white_fg_red_bg }}    2    "
+          ansible_callback_diy_playbook_on_task_start_msg: "\ndiy output(via task vars): task example: {{ ansible_callback_diy.task.name }}"
+          ansible_callback_diy_playbook_on_task_start_msg_color: bright magenta
+          ansible_callback_diy_runner_on_ok_msg: "diy output(via task vars): result example: \n{{ ansible_callback_diy.result.output.msg }}\n"
+          ansible_callback_diy_runner_on_ok_msg_color: "{{ 'yellow' if ansible_callback_diy.result.is_changed else 'bright green' }}"
+
+      - name: suppress output
+        debug:
+          msg: i should not be displayed
+        vars:
+          ansible_callback_diy_playbook_on_task_start_msg: ""
+          ansible_callback_diy_runner_on_ok_msg: ""
+
+      - name: using alias vars (see ansible.cfg)
+        debug:
+          msg:
+        when: False
+        vars:
+          ansible_callback_diy_playbook_on_task_start_msg: ""
+          on_skipped_msg: "diy output(via task vars): skipped example:\n\e[0m\e[38;5;4m\u25b6\u25b6 {{ ansible_callback_diy.result.task.name }}\n"
+          on_skipped_msg_color: white
+
+      - name: just stdout
+        command: echo some stdout
+        vars:
+          ansible_callback_diy_playbook_on_task_start_msg: "\n"
+          ansible_callback_diy_runner_on_ok_msg: "{{ ansible_callback_diy.result.output.stdout }}\n"
+
+      - name: multline output
+        debug:
+          msg: "{{ multiline }}"
+        vars:
+          ansible_callback_diy_playbook_on_task_start_msg: "\ndiy output(via task vars): task example: {{ ansible_callback_diy.task.name }}"
+          multiline: "line\nline\nline"
+          ansible_callback_diy_runner_on_ok_msg: |+2
+            some
+            {{ ansible_callback_diy.result.output.msg }}
+            output
+
+          ansible_callback_diy_playbook_on_task_start_msg_color: bright blue
+
+      - name: indentation
```

Comment:
```suggestion
      - name: Indentation
```
and so on for all `-name: Must start with a capital letter`